### PR TITLE
WW-2655: OPERA Surface Water Extent story

### PR DESCRIPTION
**Pull request by minniewong**
## Description

Fixes #WV-2655.

Tour Story about the OPERA Surface Water Extent layer

## How To Test

1. `npm run build && npm start`
2. Open with [these URL parameters](http://localhost:3000/?v=-174.2584033868648,-77.00385795537836,141.53238355885276,78.40861436211223&l=Coastlines_15m,Reference_Features_15m,Reference_Labels_15m,OPERA_L3_Dynamic_Surface_Water_Extent-HLS_Provisional(disabled=5),HLS_L30_Nadir_BRDF_Adjusted_Reflectance(hidden),HLS_S30_Nadir_BRDF_Adjusted_Reflectance(hidden),Land_Water_Map&lg=true&tr=surface_water_extent&t=2023-09-16-T19%3A28%3A37Z). *Note that the tour story is hidden from the "Introduction to Worldview" tour story window, so it cannot be accessed from there.
3. Go through the story and make sure the text and map steps look good.

## PR Submission Checklist

This is simply a reminder of what we are going to look for before merging your code.

- I have read the [CONTRIBUTING](https://github.com/nasa-gibs/worldview/blob/main/.github/CONTRIBUTING.md) doc
- I have added necessary documentation (if applicable)
- I have added tests that prove my fix is effective or that my feature works (if applicable)
- Any dependent changes have been merged and published in downstream modules (if applicable)

## Merging

Please use the `squash and merge` commit method unless each commit in your branch is vital to the commit history of main.

@nasa-gibs/worldview

### CODE REVIEW

File: config/default/common/config/metadata/stories/surface_water_extent/step005.md [hunk 1]
```diff
@@ -0,0 +1 @@
+The Indus River Valley receives heavy monsoon rains each summer which contribute to the apperance of extensive surface water. These false-color reflectance images from Landsat 8 show water in dark blues. The left image is from 3 May 2023, before the monsoon rains, and the right side is from 7 August 2023. Swiping the bar back and forth reveals the extent of floodwaters along the riverbanks and in the surrounding region.
```
Comment (Tomcariello) [inline config/default/common/config/metadata/stories/surface_water_extent/step005.md:1]: "appearance" is misspelled.

It seems to me that the left image (prior to the monsoon in the description) is "wetter" than the right side based on the darker blue (open water) in the images. Am I just confused or can the description be improved?

File: config/default/common/config/metadata/stories/surface_water_extent/step007.md [hunk 2]
```diff
@@ -0,0 +1,2 @@
+The Kakhovka Dam, along the Dnieper River in Ukraine was breached on 6 June 2023. The left image is from 5 June 2023, and shows the filled Kahovka Reservoir in the right portion of the map. The right image is from 5 July 2023 and shows the drained reservoir. Turn on and off the OPERA surface water extent layer to see that the reservoir is no longer full of water, but is mostly brown, exposed ground.
```
Comment (Tomcariello) [inline config/default/common/config/metadata/stories/surface_water_extent/step007.md:1]: "Kakhovka Dam" & "Kahovka Reservoir" have Kakhovka/Kahovka spelled differently.

**Review by Tomcariello — CHANGES_REQUESTED** [review]
This looks great aside from the couple of typos.

File: config/default/common/config/metadata/stories/surface_water_extent/step005.md (hunk 1 above)
Comment (minniewong) [reply config/default/common/config/metadata/stories/surface_water_extent/step005.md]: Fixed the typo. I added the wrong link for step 5, so I've updated that now!

**Review by Tomcariello — APPROVED** [review]

**Review by ryanweiler92 — APPROVED** [review]

**Pull request merged**